ldaptive/src/main/java/org/ccci/idm/user/ldaptive/dao/util/DnUtils.java

```diff
@@ -0,0 +1,44 @@
+package org.ccci.idm.user.ldaptive.dao.util;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Lists;
+import org.ccci.idm.user.Dn;
+import org.ldaptive.DnParser;
+import org.ldaptive.LdapAttribute;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public class DnUtils {
+    private static final String DELIMITER = ",";
+    private static final String VALUE_DELIMITER = "=";
+
+    @Nonnull
+    public static Dn parse(@Nullable final String rawDn) {
```

Comment:
wasn't totally happy with the names of these 2 util methods.

thoughts on `toGroup` & `toString` instead?